BUILDING.md

```diff
@@ -300,53 +300,55 @@ To install this version of Node.js into a system directory:
 
 To verify the build:
 
-```console
-$ make test-only
+```text
+make test-only
 ```
 
 At this point, you are ready to make code changes and re-run the tests.
 
-If you are running tests before submitting a pull request, the recommended
-command is:
+If you are running tests before submitting a pull request, use:
 
-```console
-$ make -j4 test
+```text
+make test
```

Comment:
Compiling takes longer without `-j4` on, for example, machines with 4 cores. Running tests will not. There is only one test process (`tools/test.py`) and it handles spawning multiple concurrent processes when appropriate.

I removed it because:

1. Most other commands in the doc that also have an implicit compile-if-anything-changed step don't have it, such as `make install` and `make coverage`.
2. Leaving it out here makes the important part of the command more clear.
3. There is already a statement telling people to run `make -j4` after they change `src` and `lib` files.

All that said, I can put it back in. But that's why I removed it.

Reply:
I've restored `-j4` here.